posthog/clickhouse/migrations/0037_performance_events.py

```diff
@@ -0,0 +1,21 @@
+from infi.clickhouse_orm import migrations
+
+from posthog import settings
+from posthog.models.performance.sql import (
+    DISTRIBUTED_PERFORMANCE_EVENTS_TABLE_SQL,
+    KAFKA_PERFORMANCE_EVENTS_TABLE_SQL,
+    PERFORMANCE_EVENTS_TABLE_MV_SQL,
+    PERFORMANCE_EVENTS_TABLE_SQL,
+    WRITABLE_PERFORMANCE_EVENTS_TABLE_SQL,
+)
+
+operations = [
+    migrations.RunSQL(PERFORMANCE_EVENTS_TABLE_SQL()),
+    migrations.RunSQL(KAFKA_PERFORMANCE_EVENTS_TABLE_SQL()),
+    migrations.RunSQL(PERFORMANCE_EVENTS_TABLE_MV_SQL()),
+]
+if settings.CLICKHOUSE_REPLICATION:
```

Comment:
Let's remove this if-else. This _will_ cause more issues than it solves.

Reply:
OK, so we don't add clickhouse replication here 👍... do I need to account for it in table names in the sql file?

Reply:
Please leave everything else as-is.